userguide/sharing.rst

```diff
@@ -2168,19 +2168,22 @@ and extents.
 .. table:: Associated Target Configuration Settings
    :class: longtable
 
-   +-------------+----------------+--------------------------------------------------------+
-   | Setting     | Value          | Description                                            |
-   |             |                |                                                        |
-   +=============+================+========================================================+
-   | Target      | drop-down menu | Select an existing target.                             |
-   |             |                |                                                        |
-   +-------------+----------------+--------------------------------------------------------+
-   | LUN ID      | integer        | Select or enter a value between *0* and *1023*.        |
-   |             |                | Some initiators expect a value less than *256*.        |
-   +-------------+----------------+--------------------------------------------------------+
-   | Extent      | drop-down menu | Select an existing extent.                             |
-   |             |                |                                                        |
-   +-------------+----------------+--------------------------------------------------------+
+   +-------------+----------------+-----------------------------------------------------------+
+   | Setting     | Value          | Description                                               |
+   |             |                |                                                           |
+   +=============+================+===========================================================+
+   | Target      | drop-down menu | Select an existing target.                                |
+   |             |                |                                                           |
+   +-------------+----------------+-----------------------------------------------------------+
+   | LUN ID      | integer        | Select or enter a value between *0* and *1023*. Some      |
+   |             |                | initiators expect a value less than *256*. Each target    |
+   |             |                | must have a unique :guilabel:`LUN ID`, but associated     |
```

Comment:
"Use unique LUN IDs for each associated target."

Reply:
"associated targets with unique targets" doesn't sound quite right.
How about something like "LUN IDs must be unique to each target"  ?

Reply:
Wait, that's what wblock is saying.